### ISSUE TITLE
is there a serious over-fitting?

### ISSUE THREAD

**Issue by yangz0626** [xingyizhou/CenterNet#160]
When I train ddd_3dop, I'm surprised to find that val loss becoming larger and larger while training, And model_last.pth is better than model_best.pth
For example
| epoch | train loss | val loss |
| :------| ------: | :------: |
| 5 | 2.24 | 2.89 |
| 10 | 1.86 | 2.88 |
| 15 | 1.77 | 2.90 |
| 20 | 1.69 | 3.08 |
| 25 | 1.63 | 3.08 |
| 30 | 1.60 | 3.06 |
| 35 | 1.58 | 3.07 |
| 40 | 1.56 | 3.18 |
| 45 | 1.55 | 3.26 |
| 50 | 1.45 | 3.36 |
| 55 | 1.42 | 3.50 |
| 60 | 1.41 | 3.57 |
| 65 | 1.41 | 3.60 |
| 70 | 1.41 | 3.60 |
This situation is even much more severe when using my own task(3d detecion) with this arch. val loss is 10 times and more larger than b, val loss become larger just in epoch 5......
Why it happended and how to solve it using CenterNet model?

**Comment by xingyizhou** [comment]
See https://github.com/xingyizhou/CenterNet/issues/148. Loss is not aligned well with AP. You can plot your AP to see if it is overfitting.
